### DISCUSSION TITLE
Helix Discord

### DISCUSSION THREAD

**Discussion by ghost** [helix-editor/helix#11535] [Ideas] [answered]
Hello everyone I started a Helix discord here if you wanted to join so we could all stay up to date and learn from each other! 

https://discord.gg/TmhHeqSWav

**Comment by ghost** [comment]
It would be great to get some official helix devs involved! :-) I would love to add updates and announcements to the discord for people to see new things about Helix.

**Comment by the-mikedavis** [comment, marked as answer]
Note that this is unofficial, also see https://github.com/helix-editor/helix/discussions/859#discussioncomment-1488274
